frontend/src/components/Order/components/ChangeDataBtn/ChangeDataBtn.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '../../../ui-kit/components/Button';
+import { ICONS } from '../../../ui-kit/icons';
+import styles from './ChangeDataBtn.scss';
+
+const ChangeDataBtn = ({ sectionState, handleChangeData }) => {
+  const initialState = sectionState;
```

Comment:
Why do you need that? You wanted just rename it?

Reply:
I don't use component ChangeDataBtn. I deleted it.